modules/gsp-ci/secrets.tf

```diff
@@ -1,3 +1,12 @@
+resource "random_string" "concourse_username" {
```

Comment:
I'm not sure having a random username is really adding much in the way of security (consider some kind of "user x did y" message appearing a publicly exposed dashboard UI somewhere) .... and I think it may be confusing if we saw "user Xf7dts3 did y" in an audit log or whatever

... since the only thing that should ever be using this localUser account is the pipeline-controller ... maybe just use "pipeline-controller" as the username?

Reply:
Fair.

Eliminating a static user, gives less potential to match password :trollface: and we know it's only ever pipeline-operator's user :P